PurchasesCoreSwiftTests/TestHelpers/ContainerFactory.swift

```diff
@@ -68,6 +68,18 @@ class ContainerFactory {
                              internalContainers: [])
     }
 
+    func int64Container(int64: Int64) -> ASN1Container {
+        let intAsBytes = int64ToBytes(int: int64)
+        let bytesUsedForLength = intAsBytes.count < 128 ? 1 : intToBytes(int: intAsBytes.count).count + 1
+
+        return ASN1Container(containerClass: .application,
+                             containerIdentifier: .octetString,
+                             encodingType: .primitive,
+                             length: ASN1Length(value: intAsBytes.count, bytesUsedForLength: bytesUsedForLength),
+                             internalPayload: ArraySlice(intAsBytes),
+                             internalContainers: [])
+    }
```

Comment:
this is the same as `intContainer`, but with 64. Maybe it's because I'm really tired on a Friday, but I couldn't think of a much better way to do this avoiding duplication. Open to ideas.

Reply:
same for the other changes in this class

Reply:
I think it's fine. It's more clear like this and they are not huge functions